client/amplify/backend/auth/clientbec096a1/clientbec096a1-cloudformation-template.yml

```diff
@@ -0,0 +1,433 @@
+AWSTemplateFormatVersion: 2010-09-09
+
```

Comment:
Hey i think you should not upload those files and add them to the git ignore 

as already they are tracked you should remove those files from git with `git rm `